content/blog/harbor-2.7.md

```diff
@@ -0,0 +1,124 @@
+* Add jobservice monitoring api list pool, worker and stop running task by @stonezdj in [17658](https://github.com/goharbor/harbor/pull/17658)
+* feat: support customize session timeout by @chlins in [17767](https://github.com/goharbor/harbor/pull/17767)
+* Add REST API to list job queue status, pause/resume job queue and list schedulers by @stonezdj in [17707](https://github.com/goharbor/harbor/pull/17707)
+* Add Jobservice UI by @AllForNothing in [17722](https://github.com/goharbor/harbor/pull/17722)
+
+
+The job service dashboard is a web-based interface that allows you to view and manage jobs that are running in the Harbor job service. It is available at `https://<harbor_url>/harbor/job-service-dashboard/`. You can use the dashboard to view the status of job queues, the schedule of jobs, and the status of the job service pools and workers. you can also use it to stop pending or running jobs, or pause or resume the job service queue.
+#### View Job Service Queue Status
+
+1. Log in to the Harbor Web interface with an account that has Harbor system administrator privileges.
+1. Expand **Administration**, and select **Job Service Dashboard**.
+
+In the job service dashboard, you can view the status of the job service queue, the job service pools, and the job service workers.
+
+![Job service dashboard](../img/blog-2.7/jobservice-dashboard.png)
+
+The following table describes the information that is displayed in the dashboard.
+
+|Field|Description|Actions|
+|:---|:---|:---|
+|**Pending Jobs In Queues**|The job service queue is a FIFO queue that stores jobs to be executed. Total displays the count of queue type, and the top 2 queues| **Stop All**: stop all jobs in all queues |
+|**Schedules**|The job service pool is a pool of workers that execute jobs, display the total count of schedules and show its status | **Pause All**: Pause all running job schedule, **Resume All**: Resume all paused job schedule |
+|**Workers**| The job service worker is a goroutine that executes jobs, it displays the free/total workers| **Free All**: stop the execution of running jobs to free all workers |
+
+## View Job Queue Details
+
+In the job service dashboard, click the **Job Queues** tab.
+
+![Job queues](../img/blog-2.7/jobqueues.png)
+
+The **Job Queues** tab displays the detail of job queues. The following table describes the information that is displayed in the **Job Queues** tab.
+
+|Field|Description|
+|:---|:---|
+|**Job Type**|The name of the job queue.|
+|**Pending Count**|The number of jobs that are waiting to be executed in the queue.|
+|**Latency**|The waiting time of the current queue, it indicate how long tasks in the queue have already waited.|
+|**Paused**|The paused status of job queue.|
+
+Full documentation [here](https://goharbor.io/docs/edge/administration/jobservice-dashboard/)  
+
+### Replication by chunk
+Support copy over chunk when copying image blobs for harbor replication.
```

Comment:
can we add more on chunk based replication? it fits for the edge scenario which was often asked at CNCF slack.

`For some users, they need to replicate images from a central port to an edge port, but often the network at the edge is limited or even unstable. Low bandwidth and high latency environments are the biggest challenges for replication.` ...